docs/developers/contributing.md

```diff
@@ -23,7 +23,12 @@ Set the `upstream` remote to the base `napari` repository:
 git remote add upstream https://github.com/napari/napari.git
 ```
 
-Install the package in editable mode, along with all of the developer tools
+Napari can be installed on most macOS, Linux, and Windows systems with 
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org) by
```

Comment:
```suggestion
![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg)] by
```

I think that the link to python org is obsolete here:

![Zrzut ekranu z 2022-07-17 22-10-35](https://user-images.githubusercontent.com/3826210/179423174-281176ca-067b-4f23-a36a-167242f74bc8.png)